project/{{ cookiecutter.__folder_name }}/devops/README-GITLAB.md

```diff
@@ -39,16 +84,15 @@ See [Use SSH keys to communicate with GitLab](https://docs.gitlab.com/ee/user/ss
 Check the section "Server Setup" in the file `README.md` in the same folder as this file for details.
 Be aware that the key generated by this template will not be added to your git repository because it is explicitly excluded by the `.gitignore` file.
 
-## Automatic deployment 🚀
+## Development & deployment 🚀
 
-The deployment is executed automatically on each commit to `main` branch.
+The development should be done in a separate branch such as `develop`.
 
-You may want to adapt your development workflow to that, and use a `develop` branch to develop and create merge requests for deployments.
+When opening a merge requests from that branch to `main`, the checks and tests will be executed and docker images
```

Comment:
Please verify that the pipeline logic in this sentence is correct. I assume that if either checks or tests fail, then the Docker images will not be built.

```suggestion
When opening a merge request from your development branch to `main`, CI/CD runs the checks and tests, and, if they pass, then builds the Docker images.
```